github_action_resources/entrypoint.sh

```diff
@@ -30,6 +30,7 @@ fi
 [[ -n "$INPUT_SOFT_FAIL_ON" ]] && SOFT_FAIL_ON_FLAG="--soft-fail-on $INPUT_SOFT_FAIL_ON"
 [[ -n "$INPUT_HARD_FAIL_ON" ]] && HARD_FAIL_ON_FLAG="--hard-fail-on $INPUT_HARD_FAIL_ON"
 [[ -n "$INPUT_REPO_ROOT_FOR_PLAN_ENRICHMENT" ]] && INPUT_REPO_ROOT_FOR_PLAN_ENRICHMENT_FLAG="--repo-root-for-plan-enrichment $INPUT_REPO_ROOT_FOR_PLAN_ENRICHMENT"
+[[ -n "$INPUT_DEEP_ANALYSIS" ]] && INPUT_DEEP_ANALYSIS_FLAG="--deep-analysis $INPUT_DEEP_ANALYSIS"
```

Comment:
I believe that the flag is just `--deep-analysis`, not `--deep-analysis true`, so it should look like the ones below (e.g. `--output-bc-ids`):

```
if [ -n "$INPUT_DEEP_ANALYSIS_FLAG" ] && [ "$INPUT_DEEP_ANALYSIS_FLAG" = "true" ]; then
  INPUT_DEEP_ANALYSIS_FLAG ="--deep-analysis"
fi
```

Reply:
You're absolutely right. Fixing now.